config/install/core.entity_view_display.media.gallery.default.yml

```diff
@@ -0,0 +1,102 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.gallery.field_slide
+    - media_entity.bundle.gallery
+  module:
+    - slick_media
+id: media.gallery.default
+targetEntityType: media
+bundle: gallery
+mode: default
+content:
+  field_slide:
+    weight: 0
+    label: hidden
+    settings:
+      vanilla: true
+      optionset: default
+      optionset_thumbnail: ''
+      skin: ''
+      skin_thumbnail: ''
+      image_style: slick_media
```

Comment:
Do we have to call this image style "slick_media". I would expect something that would suggest a gallery item.